docs/examples/pageSizeChanger.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Pagination from 'rc-pagination';
+import Select from 'rc-select';
+import '../../assets/index.less';
+
+export default () => {
+  const pageSizeOnChange = (size) => {
+    console.log({ size });
+  };
+
+  return (
+    <>
+      <Pagination
+        defaultCurrent={1}
+        total={50}
+        selectComponentClass={Select}
+        showSizeChanger
+        pageSizeChanger={{
+          options: [10, 25, 50, 75, 100],
+          showSearch: false,
+          onChange: pageSizeOnChange,
+        }}
```

Comment:
```suggestion
        showSizeChanger={{
          options: [10, 25, 50, 75, 100],
          showSearch: false,
          onChange: pageSizeOnChange,
        }}
```

We could reuse the `showSizeChanger` prop for accepting object value too.

Reply:
@afc163 @MadCcc  PR review changes updated. Could you please review these changes?

Reply:
@MadCcc @afc163 The test case coverage failed issue has been fixed and updated on this PR. Could you please review this PR?